docs/cluster-management/ec2-instances.md

```diff
@@ -0,0 +1,59 @@
+# EC2 instances
```

Comment:
We can link this file when we prompt for instance_type: in `clusterconfig.go, InstallPrompt function`.

e.g.

```
aws region [us-west-2]: 

s3 bucket [cortex-aa6b1c3ac8]: 

Guidelines for choosing an instance type can be found at cortex.dev/cluster-management/ec2-instances
aws instance type [m5.large]:
```